lib/models/car.js

```diff
@@ -1,4 +1,42 @@
+const pool = require('../utils/pool');
+
 class Car {
+    id;
+    make;
+    model;
+    year;
+constructor(row) {
+    this.id = row.id;
+    this.make = row.make;
+    this.model = row.model;
+    this.year = row.year;
+}
+static async insert({ make, model, year }) {
+    const { rows } = await pool.query('INSERT INTO cars (make, model, year) VALUES ($1, $2, $3) RETURNING *', [make, model, year]);
+    return new Car(rows[0]);
+}
+
+static async vallet(id) {
+    const { rows } = await pool.query('SELECT * FROM cars WHERE id = ($1)', [id]);
+    return new Car(rows[0]);
+}
+
+static async getAllCars() {
+    const { rows } = await pool.query('SELECT * FROM cars');
+    return rows.map(row => {
+        return row;
```

Comment:
Super close here! You got the map correct, but instead of returning just the Row, you want to return a new instance of the Car class. Just like you did on the other methods, only this time you'd omit the `[0]` from the end of rows. Other than that, great job!